### PR TITLE
Explain slots left command in DG

### PR DESCRIPTION
**Pull request by logical-1985516**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -209,6 +209,18 @@ The following sequence diagram shows how a search operation goes through the var
 Alternatives:
 `SearchPersonDescriptor` may be omitted - in which case, `SearchCommandParser#parse(String)` can directly use the attributes it has parsed to create a `SearchPredicate` object with the relevant attributes and information to search for. However, the methods to test `SearchPersonDescriptor` are readily available, which would ease the testing process.
 
+### SlotsLeft Command
+
+Slots left command finds the remaining vacancies of a specified job at the specified index. The remaining vacancies is the number of vacancies of the job, subtracted by the number of `OFFERED` applications to that job. The slots left operation is executed as follows:
```
Comment (acekhoon) [inline docs/DeveloperGuide.md:214]: you might want to specify that "specified index" is an index of job in the list of jobs displayed in the Ui

**Review by acekhoon — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (logical-1985516) [reply docs/DeveloperGuide.md]: Nice catch, would fix

**Review by acekhoon — APPROVED** [review]
LGTM

**Pull request merged**